### ISSUE TITLE
Feature Request - respond to commands posted by discord webhooks

### ISSUE THREAD

**Issue by SubParLou** [Frederikam/ukulele#72]
I have a discord webhook i want to send commands to ukulele to control it. However, it doesn't respond. I'm guessing because it shows up as a bot and I assume it's programmed to only respond to normal users.

**Comment by freyacodes** [comment]
It's generally bad practice to respond to messages from bots or webhooks. A configuration option could be made to bypass that. 
    
    
     
      
      Message ID: ***@***.***> 
      ***@***.***": ***@***.***": "EmailMessage","potentialAction": ***@***.***": "ViewAction","target": "https://github.com/freyacodes/ukulele/issues/72","url": "https://github.com/freyacodes/ukulele/issues/72","name": "View Issue"},"description": "View this Issue on GitHub","publisher": ***@***.***": "Organization","name": "GitHub","url": "https://github.com"}}]

**Comment by SubParLou** [comment]
yes i agree it wouldn't be something most people would want, but an option in the ukulele.yml would be amazing to enable that.
